### DISCUSSION TITLE
Placing Text on mesh

### DISCUSSION THREAD

**Discussion by tunahangormus** [jnsmalm/pixi3d#94] [Q&A] [answered]
Hello,

I have a 3D model that will write editable text on it. I was able to do this by adding the plane to the model in other 3D renderers and adding the plane to the model, but I couldn't find how to do it in pixi3D. Is it possible to do something like this? 

Also at the same time, the angle of the text should change according to the angle of the model. For example, if we see the back of the model and the text is in the front, it's like we shouldn't be able to see the text either.

Thanks.

**Comment by tunahangormus** [comment]
For now i fixed that by using text as texture and changing texture of plane that i exported from blender.

**Comment by jnsmalm** [comment, marked as answer]
Hello! Yes, that's is one way to do it. You can also use a Sprite3D object.

```
let text = new Text("test")
text.updateText()

let sprite = model.addChild(new Sprite3D(text.texture));
sprite.z = 1.5
```

If you want the sprite to always be rotated toward camera, you can set sprite.billBoardType. https://api.pixi3d.org/classes/Sprite3D.html#billboardType
